module/app/app.go

```diff
@@ -856,3 +865,19 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 
 	return paramsKeeper
 }
+
+func (app *Gravity) registerUpgradeHandlers() {
+	app.upgradeKeeper.SetUpgradeHandler("v1.5", func(ctx sdk.Context, plan upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
```

Comment:
Q: Is `v1.5` the name of the upgrade that GRAV will use?
Nit: regardless of the value, I'd opt to put this into a `const` up top :)